styles.css

```diff
@@ -86,3 +88,324 @@ ul {
   color: #505f79;
   margin: 0 10px;
 }
+
+.seebutton {
+  height: 40px;
+  width: 40px;
+  margin-left: 1px;
+  margin-top: 40px;
+}
+
+.second {
+  width: 86%;
+  justify-content: center;
+  height: 97px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  line-height: 52px;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  margin: 60px 35px;
+}
+
+.second-header img {
+  height: 4px;
+  width: 48px;
+  margin: 0 165px;
+}
+
+.grid-container {
+  margin: 90px 10px;
+  width: 90%;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 100%;
+  border-radius: 8px;
+  gap: 20px;
+  margin-bottom: 50px;
+}
+
+.grid-item {
+  margin-left: 20px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.item {
+  width: 100%;
+  background-color: #ebf0ee;
+  height: 252px;
+  border-radius: 8px;
+}
+
+.title {
+  margin: 16px 18px;
+  width: 295px;
+  height: 57px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 28px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.tagcontainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 5px;
+  gap: 20px;
+  width: 329px;
+  height: 32px;
+}
+
+.label-project {
+  background-color: #ebf0ee;
+  color: #3a4a42;
+}
+
+.see-post {
+  margin: 30px 106px 24px 103px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+}
```

Comment:
- [ ] to completely fix your button to match the figma design kindly add the below piece of code, currently your button has borders and it isn't centered and this will cemter it and remove borders :+1:
 - `border: none;`
 - `justify-content: center;`